docs/cli-reference/ks_env_targets.md

```diff
@@ -4,24 +4,43 @@ Set module targets for an environment
 
 ### Synopsis
 
-targets
+
+The `targets` command selects one or more modules to be select by an
```

Comment:
typo `to be select`. Would it be clearer as _to be applied_ (by/in) an environment?